File: bhttp-core/src/main/java/com/ldw/bhttp/parse/MyResponse.java
package com.ldw.bhttp.parse;

/**
 * @date 2020/5/23 11:20
 */
public class MyResponse<D> {
    public int code;
    public String msg;
    public  D data;

    @Override
    public String toString() {
        return "MyResponse{" +
                "code=" + code +
                ", msg='" + msg + '\'' +
                ", data=" + data +
                '}';
    }
}

File: bhttp-core/src/main/java/com/ldw/bhttp/entry/MyResponse.kt
package com.ldw.bhttp.entry

/**
 * @date 2020/5/28 12:14
 * @user 威威君
 */
class MyResponse<T> {
    var msg: String? = null
    var code: Int? = null
    var data: T? = null

    override fun toString(): String {
        return "MyResponse(msg=$msg, code=$code, data=$data)"
    }


}